ehitchhiking-ui/src/app/shared/components/registration/registration.component.sass

```diff
@@ -0,0 +1,101 @@
+fieldset
+  border: none
+  margin: 0
+  box-shadow:  0 0 0.5rem 0.1rem rgba(221, 221, 221, 1)
+
+input
+  border: none
+  font-family: inherit
+  font-size: inherit
+  outline: none
+
+a
+  color: #999
```

Comment:
use one of the existing constants or create a new one

Do that for all colors

Reply:
fix